recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ source:
 build:
   # See https://github.com/conda-forge/mujoco-feedstock/issues/22
   skip: true  # [(aarch64 or ppc64le or osx) and python_impl == 'pypy']
```

Comment:
If you want, you can get rid of that skip and all traces of pypy/python_impl

Reply:
I plan to do that in a follow up PR, thanks!